docs/developers/overview.md

```diff
@@ -0,0 +1,32 @@
+<!--
+order: 1
+-->
+
+# Developers
```

Comment:
can you add a `{synopsis}` text block? it is used to generate a preview of the page.